LX.StaffScheduler.Api/Controllers/WeatherForecastController.cs

```diff
@@ -1,33 +0,0 @@
-using Microsoft.AspNetCore.Mvc;
-
-namespace LX.StaffScheduler.Api.Controllers
-{
-    [ApiController]
-    [Route("[controller]")]
-    public class WeatherForecastController : ControllerBase
```

Comment:
Please create new contoller related to your further tasks but without any actions toward to save /Controllers directory.